packages/core/src/store/__tests__/state-store-integration.test.ts

```diff
@@ -223,14 +223,14 @@ describe('Level data store', () => {
       pinningEndpoints: ['ipfs+context'],
       networkName: 'inmemory',
     })
-    store = await storeFactory.createPinStore()
+    store = storeFactory.createPinStore()
   })
 
   it('pins document correctly without IPFS pinning', async () => {
     const genesis = await TileDoctype.makeGenesis({ content: initialContent, metadata: { controllers, tags: ['3id'] } }, context)
     const genesisCid = await dispatcher.storeCommit(genesis)
     const docId = new DocID('tile', genesisCid)
-    const doc = await Document.create(docId, doctypeHandler, dispatcher, store, context)
+    const doc = await Document.load(docId, doctypeHandler, dispatcher, store, context, DEFAULT_WRITE_DOCOPTS)
```

Comment:
maybe we should put back `Document.create()` but as just a super-simple one-line wrapper around `Document.load()` with a different default DocOpts?